cmd/osbuild-image-tests/constants/constants.go

```diff
@@ -21,7 +21,7 @@ func GetOsbuildCommand(store, outputDirectory string, exports []string) *exec.Cm
 
 func GetImageInfoCommand(imagePath string) *exec.Cmd {
 	return exec.Command(
-		"/usr/libexec/osbuild-composer-test/image-info",
+		"/usr/bin/osbuild-image-info",
```

Comment:
Should be actually `/usr/sbin` once https://github.com/osbuild/osbuild/pull/1970 lands in osbuild :innocent:

Reply:
I should have changed it to `osbuild-image-info` anyway.